CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # AEMO Gem Changelog
 
+
 ## 0.3.0-beta1
 
 * Refactor NMI allocations
 
+## 0.2.2
```

Comment:
Hmmmmm. We'll need a maintenance/v0.2 branch to do this.

Are you ok with adding to v0.3.0? Currently at https://rubygems.org/gems/aemo/versions/0.3.0.pre.rc1

See: https://github.com/jufemaiz/aemo/pull/41/files

Difference:
https://github.com/jufemaiz/aemo/compare/v0.2.1...master